src/types/window.d.ts

```diff
@@ -17,6 +17,7 @@ declare global {
 				}) => void;
 				willShowPrivacyMessage: WillShowPrivacyMessage;
 				showPrivacyManager: () => void;
+				version: string;
```

Comment:
i wonder if this should be at the top level i.e. `window.guCmpHotFix.version`?

Reply:
Sure, that makes sense.